### PR TITLE
Use different providers in Github Action workflows matrix

### PR DESCRIPTION
**Pull request by carabasdaniel**
Depends on https://github.com/puppetlabs/puppet_litmus/pull/385

### CODE REVIEW

**Comment by carabasdaniel** [comment]
Example run: https://github.com/puppetlabs/puppetlabs-testing/actions/runs/560828493

File: moduleroot/.github/workflows/nightly.yml.erb [hunk 1]
```diff
@@ -75,6 +75,7 @@ jobs:
         buildevents step $TRACE_ID $STEP_ID $STEP_START 'Setup Test Matrix'
 
   Acceptance:
+    name: "${{matrix.platforms.label}}-${{matrix.collection}}"
```
Comment (DavidS) [inline moduleroot/.github/workflows/nightly.yml.erb:78]:
```suggestion
    name: "${{matrix.platforms.label}}, ${{matrix.collection}}"
```

**Review by DavidS — CHANGES_REQUESTED** [review]

**Review by sheenaajay — APPROVED** [review]

**Review by DavidS — APPROVED** [review]

**Comment by DavidS** [comment]
so this is only waiting for a litmus release now?

**Comment by carabasdaniel** [comment]
After the litmus release that contains the new matrix_from_metadata_v2 script, one more PDK Sync run will allow us update all our supported modules to use docker containers in the acceptance matrix.

**Pull request merged**